superset-frontend/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -120,8 +121,9 @@ class FilterBox extends React.Component {
 
   getControlData(controlName) {
     const { selectedValues } = this.state;
+    const controlsMap = getControlsInventory(controlName);
```

Comment:
Should this be `getControlsInventory(vizType)`?

Reply:
Actually, I backed this change out, to pull these controls (by key) from `controls.jsx` as it was before. This part apparently loads control configs for `druid_time_origin`, `granularity`, `granularity_sqla`, and `time_grain_sqla` which are part of `controls.jsx` and are NOT part of the controls included in `FilterBox.jsx`.

Soon, my plan (and we can talk about this if I'm not making sense here) is to make a `GlobalControls.jsx` file with the widely-shared controls, structured sort of like the `Shared_NVD3.js` and `Shared_DeckGL.jsx` files, so we can import/load objects by reference rather than a string. At that point, I'll revisit this chunk of code, thus the TODO I've added.